machine.go

```diff
@@ -225,12 +224,17 @@ func (m *Machine) kernelRelease() string {
 	var u syscall.Utsname
 	syscall.Uname(&u)
 	release := charsToString(u.Release[:])
+	modpath := "/lib/modules"
 
-	if _, err := os.Stat(path.Join("/lib/modules", release)); err == nil {
+	if _, err := os.Stat(path.Join(modpath, release)); err == nil {
 		return release
 	}
 
-	files, err := ioutil.ReadDir("/usr/lib/modules")
+	if mergedUsrSystem() {
+		modpath = "/usr/lib/modules"
+	}
+
+	files, err := ioutil.ReadDir(modpath)
```

Comment:
on a usr merged system /lib is a symlink to /usr/lib so doing a readdir on /lib/modules should always work without needing the extra determinations